native/constants/theme.ts

```diff
@@ -29,6 +29,18 @@ export const COLOR = {
   MESSAGE_BACKGROUND: '#1da1f2',
 };
 
+export const FONT_SIZE = {
+  TAB_TITLE: 14,
+  SUBHEADER: 15,
+  TEXT: 14,
+  INPUTAREA: 18,
+  ALERT: 12,
+  ITEM_TITLE: 16,
+  ITEM_SUBTITLE: 14,
+  BUTTON_DEFAULT: 18,
+  BUTTON_WITH_ICON: 10,
+};
```

Comment:
@sea-kai 
何度も修正依頼ごめん。COLORの方の命名規則と合わせるのであれば、以下の感じでもいいかも

```suggestion
export const FONT_SIZE = {
  TITLE_HEADER: 16,
  TITLE_SUBHEADER: 16,
  TEXT_DEFAULT: 12,
  TEXT_ALERT: 12,
  TEXT_INPUT: 16,
  TAB_TITLE: 16,
  ITEM_TITLE: 16,
  ITEM_SUBTITLER: 12,
  BUTTON_DEFAULT: 16,
  BUTTON_WITH_ICON: 12,
};
```

あと、このタイミングで文字サイズについてもある程度規則決めたくて..
↓のような8の倍数のサイズにするのが有名だけど、ネイティブアプリで8もサイズ差つけるとすごいことなりそうなので、4の倍数でサイズ定義してく感じで様子見してみたい

> https://yuyakinoshita.com/blog/2019/02/10/design-by-multiple-of-8/

@wf-yamaday
フォントサイズの定義、これで過不足ないかみて欲しい。
(文字サイズについては最悪、マージしてから少しずつ確認でもいいか🤔)

Reply:
コメントありがとうございます！
定義の修正、追加しておきました！